src/vs/workbench/contrib/files/browser/views/explorerView.ts

```diff
@@ -991,7 +991,10 @@ registerAction2(class extends Action2 {
 
 	run(accessor: ServicesAccessor) {
 		const viewsService = accessor.get(IViewsService);
-		const explorerView = viewsService.getViewWithId(VIEW_ID) as ExplorerView;
-		explorerView.collapseAll();
+		const view = viewsService.getViewWithId(VIEW_ID);
+		if (view !== null) {
```

Comment:
Do we want `if (view)` to cover null and undefined?

Reply:
since the type of `view` is `IView | null` I didn't think so

Reply:
(normally I would do that but because big codebase I wanted it to be an explicit check to avoid confusion)